src/components/reusable/notification/notification.scss

```diff
@@ -0,0 +1,58 @@
+@use '../../../common/scss/global.scss';
+@use '../../../common/scss/mixins.scss';
+@use '~@kyndryl-design-system/shidoka-foundation/scss/mixins/typography.scss';
+
+:host {
+  display: block;
+  width: 100%;
+  kyn-card {
+    width: 100%;
+    margin-top: 8px;
```

Comment:
Don't use margin here. Use notification-panel to create the gaps.

Reply:
ok